README.md

```diff
@@ -31,6 +31,10 @@ values required.
 
 `$ ./run-local.sh`
 
+To run start up the application without pulling / restarting local containers:
+
+`$ ./run-local.sh --skip-docker`
+
```

Comment:
love the script but it takes a while to run and tears down the DB. This flag prevents that so it just runs the app